CSharp.Homeworks/hw_15/HW15.Task2/CreditCardAttribute.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+
+
+
+namespace HW15.Task2
+{
+    [AttributeUsage(AttributeTargets.Property)]
+    internal class CreditCardAttribute: Attribute
+    {
+        public bool IsValidCard(string numCard)
+        {
+            if (string.IsNullOrEmpty(numCard))
+            {
+                return true;
+            }
+            
+            Regex regN = new(@"\d{4}([\s-]\d{4}){3}");
+            Regex regL = new(@"\d{16}");
+            if (regN.Match(numCard).Success || regL.Match(numCard).Success)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }           
```

Comment:
you can use this expression instead
`return regN.Match(numCard).Success || regL.Match(numCard).Success`

Reply:
Of course, thank you :)